about/2i2c.md

```diff
@@ -1,22 +1,54 @@
-# About 2i2c
+# Our unique qualifications and services
 
-[2i2c](http://2i2c.org) (the International Interactive Computing Collaboration) makes interactive computing more accessible and powerful for research and education. It does so by developing, designing, and managing open source interactive computing infrastructure, and by contributing to open source communities that underlie and create this infrastructure.
+```{epigraph}
+2i2c is a mission-driven non-profit with expertise in cloud infrastructure, Jupyter, open science and scholarship, and open development practices.
+```
 
-:::{admonition} Learn more
-:class: tip
+2i2c provides a **managed, customized JupyterHub service** that is tailored for research and education communities.
+We manage entirely non-proprietary, open-source tools that give their user communities the [Right to Replicate](http://2i2c.org/right-to-replicate) this infrastructure with or without 2i2c.
+As a part of this service, 2i2c also makes **upstream contributions to open-source communities** as a part of continuously operating and improving this infrastructure.
```

Comment:
I suggest slight rephrasing. "....open-source tools that ensure user communities have the [Right to Replicate]..."